editor.planx.uk/src/@planx/components/TextInput/model.ts

```diff
@@ -12,6 +12,10 @@ export enum TextInputType {
   Phone = "phone",
 }
 
+export const shortTextLimit = 120;
```

Comment:
I thought it would be more maintainable if I could have these as exported variables I can use in the below conditions and also in the `<Input>` component